airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/AirbyteMessageTracker.java

```diff
@@ -238,12 +238,8 @@ private void handleEmittedOrchestratorMessage(final AirbyteControlMessage contro
   @SuppressWarnings("PMD") // until method is implemented
   private void handleEmittedOrchestratorConnectorConfig(final AirbyteControlConnectorConfigMessage configMessage,
                                                         final ConnectorType connectorType) {
-    // TODO: Update config here
-    /**
-     * Pseudocode: for (key in configMessage.getConfig()) { validateIsReallyConfig(key);
-     * persistConfigChange(connectorType, key, configMessage.getConfig().get(key)); // nuance here for
-     * secret storage or not. May need to be async over API for replication orchestrator }
-     */
+    // Currently being persisted as part of the DefaultReplicationWorker
```

Comment:
note: we shouldn't persist the config in the airbyte message tracker - the tracker is really for stats/metadata tracking and should not have side effects

Reply:
Yep, agree! Updated the comment to no longer be a TODO 😛